redwood-layout-uiview/src/commonMain/kotlin/app/cash/redwood/layout/uiview/YogaUIView.kt

```diff
@@ -42,13 +38,12 @@ internal class YogaUIView : UIScrollView(cValue { CGRectZero }) {
   }
 
   override fun intrinsicContentSize(): CValue<CGSize> {
-    return calculateLayoutWithSize(Size(Size.Undefined, Size.Undefined)).toCGSize()
+    return calculateLayoutWithSize(CGSizeMake(Size.Undefined.toDouble(), Size.Undefined.toDouble()))
```

Comment:
`Size.Undefined.toDouble()` occurs in many places, which could be pulled out into a constant to prevent the `Float` to `Double` conversion happening in many places. I'm assuming that the compiler is smart enough to do such an optimisation though.

Reply:
Why not just use `CGSizeMake(0.0, 0.0)` here? It's standard practice to use `CGRect.zero` on iOS to signal "we don't know the frame right now, but we'll calculate it later", and it feels like we're doing something similar here.

Reply:
@veyndan Does doing this conversion open us up to any floating point number issues? My worry was `Size.Undefined == Size.Undefined.toDouble().toFloat()` might not be true.

Reply:
@veyndan @dnagler Yep, passing 0 to Yoga code, would measure its children as if it had a 0 width/height. `Size.Undefined` tells the Yoga code that container has an unbounded width/height.

Reply:
NaNs do survive both conversions since they're the same concept regardless of how many bits are dedicated to the floating point representation.